src/object/schema_system_catalog.cpp

```diff
@@ -79,8 +79,8 @@ namespace cubschema
     CT_CHARSET_NAME,			// "_db_charset"
     CT_DB_SERVER_NAME,		// "_db_server"
     CT_SYNONYM_NAME,			// "_db_synonym"
-
     CT_TRIGGER_NAME,			// "db_trigger"
+    CT_DB_HISTOGRAM_NAME,		// "_db_histogram"
```

Comment:
```suggestion
    CT_HISTOGRAM_NAME,		// "_db_histogram"
```
대부분 DB_ 를 붙여서 변수명으로 사용하고 있지 않아서요.
일관성을 위해 제거하는 것도 좋을 것 같습니다.

참고만 해주세요.